### PR TITLE
Add napari-vedo-bridge recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

---

@jaimergp, @jo-mueller do you agree with being added as maintainers?

**Comment by jo-mueller** [comment]
@goanpeca  Sure!

File: recipes/napari-vedo-bridge/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+about:
+  home: https://github.com/jo-mueller/napari-vedo-bridge
+  summary: Transfer mesh data between napari and vedo for interactive processing
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - goanpeca
+    - jaimergp
```
Comment (jo-mueller) [inline recipes/napari-vedo-bridge/meta.yaml:46]:
```suggestion
    - jaimergp
    - jo-mueller
```

**Comment by goanpeca** [comment]
Windows failure is expected. Package is noarch and some win deps are missing.
        
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-vedo-bridge/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - magicgui
+    - numpy
+    - python >=3.8
+    - qtpy
+    - vedo >=2023.4.6
+
+test:
+  imports:
+    - napari_vedo_bridge
```
Comment (jaimergp) [inline recipes/napari-vedo-bridge/meta.yaml:30]:
```suggestion
    - napari_vedo_bridge
    - napari_vedo_bridge._cutter_widget
```

`__init__` is empty

File: recipes/napari-vedo-bridge/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - napari_vedo_bridge._cutter_widget
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
```
Comment (jaimergp) [inline recipes/napari-vedo-bridge/meta.yaml:36]:
```suggestion
    - pip
    - pyqt
```

**Pull request merged**